core/src/storage/device.py

```diff
@@ -84,13 +85,23 @@ def get_rotation() -> int:
     return int.from_bytes(rotation, "big")
 
 
+def set_rotation(value: int) -> None:
+    if value not in (0, 90, 180, 270):
+        raise ValueError  # unsupported display rotation
+    common.set(_NAMESPACE, _ROTATION, value.to_bytes(2, "big"), True)  # public
```

Comment:
Is the `# public` comment anyhow better than `public=True`? It seems ugly and somewhat less readable anyway.

Reply:
good point. i'll modify all instances

Reply:
a503c07